README.md

```diff
@@ -1439,6 +1439,7 @@ See also [Are we game yet?](https://arewegameyet.rs)
 ### Parsing
 
   * [comex/rust-shlex](https://github.com/comex/rust-shlex) [[shlex](https://crates.io/crates/shlex)] — Split a string into shell words, like Python's shlex. [![build badge](https://github.com/comex/rust-shlex/actions/workflows/test.yml/badge.svg?branch=master)](https://github.com/comex/rust-shlex/actions/workflows/test.yml)
+  * [tafia/calamine](https://github.com/tafia/calamine) - An Excel/OpenDocument Spreadsheets file reader/deserializer, in pure Rust.
```

Comment:
Please add build badge if possible

Reply:
Also, seems like the order goes on the first letter, in this case the "t" in "tafia", so maybe you need to move this item below

Reply:
Making the changes